howto/install/deploy-bare-metal.md

```diff
@@ -0,0 +1,93 @@
+
+    juju bootstrap manual/<user>@<controller IP address> anbox-cloud
+
+Juju will connect to the machine via SSH as the specified user and install all necessary requirements.
+
+When the controller is set up, create a model to hold the Anbox Cloud deployment:
+
+    juju add-model main
+
+## Add all machines
+
+Before starting the deployment, you must add all machines to Juju. See [Minimum hardware](https://discourse.ubuntu.com/t/requirements/17734#minimum-hardware) for the list of machines that you need.
+
+When adding the machines, start with the machine that you want to host the management layer of Anbox Cloud. Then add all LXD worker nodes. Run the following command for each machine:
+
+    juju add-achine ssh:<user>@<machine IP address>
+
+The user (for example, `ubuntu`) must have administrator rights on the machine and have permission to SSH to the machine.
+
+[note type="caution" status="Warning"]Make sure to add the machines by their IP addresses rather than their DNS names. Adding a machine by DNS name does currently not work.[/note]
+
+Juju will add the machines to its list of usable machines, which you can display with the `juju list-machines` command. Make sure that all machines are in the `started` state before you proceed. If any of the machines are still in `down` state, wait until they switch to `started`:
+
+    Machine  State    DNS           Inst id              Series AZ Message
+    0        started  192.168.1.9   manual:192.168.1.9   bionic    Manually provisioned machine
+    1        started  192.168.1.10  manual:192.168.1.10  bionic    Manually provisioned machine
+
+## Attach your Ubuntu Advantage subscription
+
+Create an `ua.yaml` overlay file as described in [Deploy Anbox Cloud with Juju](https://discourse.ubuntu.com/t/deploy-anbox-cloud-with-juju/17744#ua-overlay).
+
+You must provide this file when deploying Anbox Cloud.
+
+## Determine the machine mapping
+
+When running the deployment command, you must map the machines to the ones described in the [Juju bundle](https://discourse.ubuntu.com/t/about-anbox-cloud/17802#juju-bundles) that you are deploying.
+
+Run `juju list-machines` to display the available machines:
+
+    Machine  State    DNS            Inst id              Series  AZ             Message
+    0        started  192.168.0.9   i-09a2fdb5e7a2e8385   bionic  localhost-1a   running
+    1        started  192.168.0.10  i-00a05065e2768be5d   bionic  localhost-1b   running
+
+The `anbox-cloud-core` deployment bundle requires two machines: `0` and `1`. `0` is supposed to host the AMS service. `1` is used for LXD. See the [bundle.yaml](https://api.jujucharms.com/charmstore/v5/~anbox-charmers/bundle/anbox-cloud-core/archive/bundle.yaml) file for details.
+
+The `anbox-cloud` bundle requires two additional machines to host the extra services required for streaming. See the [bundle.yaml](https://api.jujucharms.com/charmstore/v5/~anbox-charmers/bundle/anbox-cloud/archive/bundle.yaml) file for details.
```

Comment:
Why two additional machines? The bundle file has '0', '1' and '2'?

Reply:
You're right, we should say that it requires three machines, although this is not technically required, it's good for production environments

Reply:
The doc is currently wrong, it should read
```suggestion
The `anbox-cloud` bundle requires an additional machine to host the extra services required for streaming. See the [bundle.yaml](https://api.jujucharms.com/charmstore/v5/~anbox-charmers/bundle/anbox-cloud/archive/bundle.yaml) file for details.
```